src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -1,26 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../../build-common/NHibernate.props" />
-
```

Comment:
Should I revert those riders glitches, or just let it do that once and hopefully for all?